react/Alerter/index.jsx

```diff
@@ -22,7 +97,11 @@ class Alerter extends Component {
 
       this.closeTimer = setTimeout(() => {
         this.setState({ hidden: true })
-        if (this.props.onClose && typeof this.props.onClose === 'function') this.props.onClose()
+        if (this.props.onClose && typeof this.props.onClose === 'function') {
+          this.dismissTimer = setTimeout(() => {
```

Comment:
Because of the fade-out animation: we need to wait after the transition ended before firing the dismiss. If not, the alert would disappear immediately